Demo/Demo/ViewModels/PayPalWebViewModel.swift

```diff
@@ -10,15 +10,41 @@ class PayPalWebViewModel: ObservableObject, PayPalWebCheckoutDelegate {
 
     let configManager = CoreConfigManager(domain: "PayPalWeb Payments")
 
-    func createOrder(amount: String, selectedMerchantIntegration: MerchantIntegration, intent: String) async throws {
+    func createOrder(
+        amount: String,
+        selectedMerchantIntegration: MerchantIntegration,
+        intent: String,
+        shouldVault: Bool
+    ) async throws {
         // might need to pass in payee as payee object or as auth header
 
         let amountRequest = Amount(currencyCode: "USD", value: amount)
         // TODO: might need to pass in payee as payee object or as auth header
+
+        var vaultPayPalPaymentSource: VaultPayPalPaymentSource?
+        if shouldVault {
+            let attributes = Attributes(
+                vault: Vault(
+                    storeInVault: "ON_SUCCESS",
+                    usageType: "MERCHANT",
+                    customerType: "CONSUMER"
+                )
+            )
+            // The returnURL is not used in our mobile SDK, but a required field for create order with PayPal payment source.
```

Comment:
Nitpick - I would also add a reference to the JIRA ticket # that summarizes this issue we need to take up with the API team!

Reply:
addressed here: c77464c24fd783711665df8cbc5e07f6f0d09fd2